app/views/reservations/new.html.haml

```diff
@@ -21,8 +21,8 @@
 
 .wysiwyg= @instrument.description
 
-- unless @instrument.online?
-  %p.alert.alert-danger= t(".offline_notice")
+- if @instrument.offline? || @instrument.offline_via_schedule_share?
```

Comment:
If we stick with this concept, I'm open to a new name. Most of the alternatives I could think of were longer.